dist/radar_client.js

```diff
@@ -564,10 +580,14 @@ var init = function(name) {
   };
 };
 
-for(var i = 0; i < props.length; i++){
+for (var i = 0; i < props.length; i++) {
   init(props[i]);
 }
 
+Scope.prototype._buildScopePrefix = function (typeName, scope, accountName) {
+  return typeName + ':/' + accountName + '/' + scope;
+};
+
```

Comment:
If you are referring to what appears to be *new* code in dist/radar_client.js, the truth of the matter is that for radar_client, dist/ fell behind lately, due to my efforts to limit the surface area of change in these PRs.  The best solution is to generate dist/radar_client.js on *master* only, and to build it with a script prior to publishing the latest radar_client to NPM.